assets/src/setup/pages/save/index.js

```diff
@@ -212,15 +212,15 @@ export function Save() {
 									target="_blank"
 									rel="noreferrer"
 								>
-									{ __( 'Visit in Customizer', 'amp' ) }
+									{ __( 'Customize', 'amp' ) }
```

Comment:
Same reasoning as above. Maybe it could instead be something like "Customize AMP page".

Reply:
If not just “Browse” and “Customize” per my comment above, what about “Browse AMP” and “Customize AMP”?

Reply:
Yea that works as well.

Reply:
Updated in b2661a832